react/features/recording/components/Recording/web/StartRecordingDialogContent.tsx

```diff
@@ -0,0 +1,429 @@
+/* eslint-disable lines-around-comment  */
+import React from 'react';
+
+// @ts-ignore
+import { translate } from '../../../../base/i18n';
+// @ts-ignore
+import {
+    Container,
+    Image,
+    LoadingIndicator,
+    Switch,
+    Text
+    // @ts-ignore
+} from '../../../../base/react';
+// @ts-ignore
+import { connect } from '../../../../base/redux';
+import Button from '../../../../base/ui/components/web/Button';
+import { BUTTON_TYPES } from '../../../../base/ui/constants';
+// @ts-ignore
+import { RECORDING_TYPES } from '../../../constants';
+// @ts-ignore
+import { getRecordingDurationEstimation } from '../../../functions';
+import AbstractStartRecordingDialogContent, {
+    type Props,
+    mapStateToProps
+} from '../AbstractStartRecordingDialogContent';
+import {
+    DROPBOX_LOGO,
+    ICON_CLOUD,
+    ICON_INFO,
+    ICON_USERS,
+    LOCAL_RECORDING,
+    TRACK_COLOR
+    // @ts-ignore
+} from '../styles.web';
+
+
+/**
+ * The start recording dialog content for the mobile application.
+ */
+class StartRecordingDialogContent extends
+// @ts-ignore
+    AbstractStartRecordingDialogContent<Props> {
+    /**
+     * Renders the component.
+     *
+     * @protected
+     * @returns {React$Component}
+     */
+    render() {
+        return (
+            <Container className = 'recording-dialog'>
+                { this._renderNoIntegrationsContent() }
+                { this._renderFileSharingContent() }
+                { this._renderUploadToTheCloudInfo() }
+                { this._renderIntegrationsContent() }
+                { this._renderLocalRecordingContent() }
+            </Container>
+        );
+    }
+
+    _shouldRenderNoIntegrationsContent: () => boolean;
+
+    _onRecordingServiceSwitchChange: () => void;
+
+
+    /**
+     * Renders the content in case no integrations were enabled.
+     *
+     * @returns {React$Component}
+     */
+    _renderNoIntegrationsContent() {
+        if (!this._shouldRenderNoIntegrationsContent()) {
+            return null;
+        }
+
+        const {
+            _localRecordingEnabled,
+            integrationsEnabled,
+            isValidating,
+            isVpaas,
+            selectedRecordingService,
+            t
+            // @ts-ignore
```

Comment:
Ditto

Reply:
Property 'props' does not exist on type 'StartRecordingDialogContent'